account_invoice_change_currency/models/account_change_currency.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2017-2018 Vauxoo
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import api, fields, models
+from odoo.tools import float_compare
+
+
+class AccountMoveLine(models.Model):
```

Comment:
Move below `AccountMove`, as this one is not the primary class

Reply:
Done!